torchao/prototype/safetensors/safetensors_support.py

```diff
@@ -14,7 +14,13 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-def load_tensor_state_dict(file_path: str, device: str):
+def load_tensor_state_dict(
```

Comment:
also for this one, I feel it might be better to have this one just accept `loaded_tensors` and `metadata` and do the `load_file` outside of this function